pkgs/by-name/de/delugia-code/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
```

Comment:
consider using [fixed-point arguments](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes) with `mkDerivation` instead of `rec`.